src/pages/Introduction/Introduction.scss

```diff
@@ -1,5 +1,6 @@
 .introduction {
```

Comment:
[via ChatGPT review bot] Noticed that you've added a background color directly here in the scss file. It's generally recommended to define colors in a separate central location where all colors are defined. This will make your code more maintainable and it's easier to make changes since you only have to edit one file.

```scss
$color-background: #1c1031;

.introduction {
    height: 100vh;
    background-color: $color-background;
}
```
You can check [Sass Guidelines](https://sass-guidelin.es/) for an in-depth guide on how to organize your Sass projects.